manifests/gcp_marketplace/schema.yaml

```diff
@@ -122,8 +122,16 @@ properties:
       Using CloudSQL and GCS provides better reliability and performance,
       as well as features such as data backup and usage monitoring.
       This is the recommended option especially for production scenarios.
-      If false, the data will be stored in GCE Persistent Disk.
+      If false, the data will be stored in GCE Persistent Disks.
     default: false
+  managedstorage.gcsBucketName:
+    type: string
+    title: Artifact storage GCS bucket (Managed storage only)
+    description: |-
+      This field must be specified if choose to use managed storage.
```

Comment:
Do we still need the managed storage checkbox? Doesn't the gcsBucketName replace it?

Reply:
The UX @rmgogogo and I agreed on is to still only allow all managed or all in cluster, because of limited bandwidth before release. Partial solutions are doable technically, but they are harder to document and support.

Reply:
mention when reinstall KFP, cloud SQL and this field should be well matched?

Reply:
Good point, will add that

Reply:
added description